src/Env/RecursiveSetEnvParametersTask.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace BestIt\Mage\Tasks\Env;
+
+use Mage\Task\Exception\ErrorException;
+use RecursiveDirectoryIterator;
+use RecursiveIteratorIterator;
+use SplFileInfo;
+
+/**
+ * Extends SetEnvParametersTask by a recursive directory iteration to edit all dist files with env parameters.
+ *
+ * @package BestIt\Mage\Tasks\Env
+ * @author Johannes Borgwardt <johannes.borgwardt@bestit-online.de>
+ */
+class RecursiveSetEnvParametersTask extends SetEnvParametersTask
+{
+    /**
+     * File extension string for dist files.
+     */
+    const DIST_FILE_EXTENSION = '.dist';
+
+    /**
+     * Get the Name/Code of the Task
+     *
+     * @return string
+     */
+    public function getName()
+    {
+        return 'env/recursive-set-env-parameters';
+    }
+
+    /**
+     * Get a short Description of the Task
+     *
+     * @return string
+     */
+    public function getDescription()
+    {
+        return '[Env] Set parameters from env variables in dist files recursively.';
+    }
+
+    /**
+     * Executes the Command
+     *
+     * Actually a DirectoryIterator is used instead of a GlobIterator. As the GlobIterator does not support
```

Comment:
I wouldn't actually add this as a comment, as it doesn't really matter or add beneficial information